R/g.part4.R

```diff
@@ -888,14 +888,18 @@ g.part4 = function(datadir = c(), metadatadir = c(), f0 = f0, f1 = f1,
                   nightsummary[sumi, 22] = acc_wakeTS
                   #----------------------------------------------
                   nightsummary[sumi, 23] = tmp1  #guider_onset_ts
-                  nightsummary[sumi, 24] = tmp4  #guider_onset_ts
+                  nightsummary[sumi, 24] = tmp4  #guider_wake_ts
                   if (params_sleep[["sleepwindowType"]] == "TimeInBed") {
-                    # If guider isa sleeplog and if the sleeplog recorded time in bed then
+                    # If guider is a sleeplog and if the sleeplog recorded time in bed then
                     # calculate: sleep latency:
                     nightsummary[sumi, 25] = round(nightsummary[sumi, 3] - nightsummary[sumi, 7],
                                                    digits = 7)  #sleeponset - guider_onset
                     # sleep efficiency:
-                    nightsummary[sumi, 26] = round(nightsummary[sumi, 14]/nightsummary[sumi, 9], digits = 5)  #accumulated nocturnal sleep / guider
+                    if (params_sleep[["relyonguider_SleepEfficiency"]]) {
```

Comment:
The term "relyonguider_SleepEfficiency" feels inaccurate because for the beginning of the night we will rely on guider (sleeplog) for latency estimates in both scenarios. For the end of the night we will rely on guider fully in the existing method and partially in the new method because the guider is used to identify wakeup time.

So, I think the distinction is not to rely on guider or now. Instead, maybe use `sleepefficiency.metric` with numbers 1, 2,... to indicate how it is calculated?

Reply:
Further, changelog version needs to refer to version 2.10-3 as 2.10-2 is already out.